LayoutTests/accessibility/mac/text-operation/text-operation-capitalize.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../../../resources/accessibility-helper.js"></script>
+<script src="../../../resources/js-test.js"></script>
+</head>
+<body>
+
+<div id="test-content">
+    <p contenteditable="true" id="text">The quick <span id="target1">brown fo</span>x jumps ov<span id="target2">er the la</span>zy dog.</p>
+    <p contenteditable="true" id="text2">TEXT2: The quick brown fox jumps over the lazy dog.</p>
+    <p contenteditable="true" id="text3">TEXT3: The quick brown fox jumps over the lazy dog.</p>
+</div>
+
+<script>
+var output = "Tests that a Capitalize text operation transforms text to capitalized as expected.\n\n";
+
+async function getMarkerRangeInElementById(id, startIndex, endIndex) {
+    const root = accessibilityController.rootElement.childAtIndex(0);
+    const selection = window.getSelection();
+    selection.removeAllRanges();
+
+    await waitFor(() => {
+        const selectedRange = root.selectedTextMarkerRange();
+        return root.textMarkerRangeLength(selectedRange) == 0;
+    });
+
+    const range = document.createRange();
+    const element = document.getElementById(id);
+    range.setStart(element.firstChild, startIndex);
+    range.setEnd(element.firstChild, endIndex);
+    selection.addRange(range);
+
+    let selectedRange;
+    await waitFor(() => {
+        selectedRange = root.selectedTextMarkerRange();
+        return root.textMarkerRangeLength(selectedRange) > 0;
+    });
+
+    return selectedRange;
+}
+
+async function waitForNotification(accessibilityElement, notification, operation) {
```

Comment:
I wanted to avoid polluting `accessibility-helpers.js`, but I do agree that it would be nice to have this function for future tests. I've moved it.

> does it have to be async?

It doesn't _need_ to be async; it was slightly more ergonomic for my specific use case, but now I'm calling it as a normal function instead because it felt more appropriate.